### ISSUE TITLE
Where are you my friend?

### ISSUE THREAD

**Issue by mpizzagalli** [francoprud/blog#1]
@francoprud are you even supporting this awesome blog? 
It would be a pity if this spectacular piece of code goes to the graveyard!

**Comment by SKOLZ** [comment]
@francoprud I agree with mpizzagalli. we could discuss new features or posts through Whats App, send me a message when you are available.
